scripts/ifAlias

```diff
@@ -85,7 +85,7 @@ alias_from_interfaces_config_file()
     if [ -x "$DISTRO_BIN" ]
     then
         case "$("$DISTRO_BIN" | cut -d " " -f 1)" in
-            Debian)
+            Debian|Ubuntu|Raspbian)
```

Comment:
Oh, whoops, looks like you need to make this valid bash.  Should probably be:
Debian(
Ubuntu)
Raspbian)

Reply:
Different error by `shellcheck`, but all are fixed ;)